### PR TITLE
Add SSL.com as a CA

### PR DESCRIPTION
**Pull request by codelux**
SSL.com is a MS trusted CA and participates in the Microsoft Root Program.

### CODE REVIEW

**Comment by JonDouglas** [comment]
Please refer to the feedback on the other pull request and we'd gladly accept your contribution.

https://github.com/NuGet/docs.microsoft.com-nuget/pull/2851

File: docs/create-packages/Sign-a-Package.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ A signed package allows for content integrity verification checks, which provide
 
 ## Get a code signing certificate
 
-Valid certificates can be obtained from a public certificate authority such as [DigiCert](https://www.digicert.com/code-signing/), [Global Sign](https://www.globalsign.com/en/code-signing-certificate/), [Comodo](https://www.comodo.com/e-commerce/code-signing/code-signing-certificate.php), or [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml). You can obtain the complete list of certification authorities trusted by Windows from [http://aka.ms/trustcertpartners](/security/trusted-root/participants-list).
+Valid certificates can be obtained from a public certificate authority such as [SSL.com](https://www.ssl.com/certificates/code-signing/), [DigiCert](https://www.digicert.com/code-signing/), [Global Sign](https://www.globalsign.com/en/code-signing-certificate/), [Comodo](https://www.comodo.com/e-commerce/code-signing/code-signing-certificate.php), or [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml). You can obtain the complete list of certification authorities trusted by Windows from [http://aka.ms/trustcertpartners](/security/trusted-root/participants-list).
```
Comment (erdembayar) [inline docs/create-packages/Sign-a-Package.md:17]: Duplicate of https://github.com/NuGet/docs.microsoft.com-nuget/pull/2851
Same PR comments regarding alphabetic ordering still applies.

**Comment by codelux** [comment]
@dotnet-policy-service agree [company=SSL.com]

**Comment by codelux** [comment]
@dotnet-policy-service agree company="SSL.com"

File: docs/create-packages/Sign-a-Package.md (hunk 1 above)
Comment (codelux) [reply docs/create-packages/Sign-a-Package.md]: the bulleted, alphabetized list can be found here https://github.com/NuGet/docs.microsoft.com-nuget/pull/3046/commits/f4b894ec23586cd5a13a75c5f49d9af4a3348a30

**Comment by codelux** [comment]
the bulleted, alphabetized PR can be found here https://github.com/NuGet/docs.microsoft.com-nuget/pull/3046/commits/f4b894ec23586cd5a13a75c5f49d9af4a3348a30

Is there any additional action on my part that needs to be done?

**Review by JonDouglas — APPROVED** [review]
Edited to keep the original language. Thank you for the contribution!

**Pull request merged**